src/main/resources/application.properties

```diff
@@ -28,6 +28,9 @@ spring.liquibase.change-log=classpath:liquibase/changelog-postgres-master.xml
 # Swagger
 spring.mvc.pathmatch.matching-strategy=ANT_PATH_MATCHER
 springdoc.swagger-ui.path=/swagger-ui
+springdoc.swagger-ui.displayOperationId=true
+springdoc.swagger-ui.tags-sorter=alpha
+springdoc.swagger-ui.operations-sorter=alpha
```

Comment:
These two lines sort the controllers in Alphabetical order in the UI.  Makes it much easier to search for an find an API.  This only affects the UI not the order the controllers are processed to generate the operation IDs.